bundles/core/org.eclipse.smarthome.core/META-INF/MANIFEST.MF

```diff
@@ -56,6 +57,11 @@ Import-Package: com.google.common.base,
  org.osgi.service.event,
  org.osgi.service.packageadmin,
  org.osgi.util.tracker,
- org.slf4j
+ org.slf4j,
+ tec.uom.lib.common,

```

Comment:
My feeling is that this would be helpful to see how we can actually make use of this new feature - and I actually expect that we learn some parts ourselves through this exercise which might lead us to doing modifications on this feature.